script.js

```diff
@@ -7,15 +8,180 @@ const menuLinks = document.querySelector('.navmenu');
+    title: 'Printing Data Two',
+    description:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    featuredImage: 'images/Background.png',
+    technologies: ['html', 'Bootstrap', 'Ruby'],
+    liveLink: 'https://have-samuel.github.io/portfolio1/',
+    sourceLink: 'https://github.com/Have-Samuel/portfolio1/tree/popup-menu',
+  },
+  {
+    projectId: 2,
+    title: 'Printing Data Three',
+    description:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    featuredImage: 'images/Background.png',
+    technologies: ['html', 'Bootstrap', 'Ruby'],
+    liveLink: 'https://have-samuel.github.io/mobile-ver/',
+    sourceLink: 'https://github.com/Have-Samuel/portfolio1/tree/popup-menu',
+  },
+];
+// Calling the cards
+const sectionCards = document.querySelector('.section-cards');
+
+function createProjectCard(project) {
+  const section = document.createElement('section');
+  section.classList = 'section1';
+  // <h4>
+  const projectHeading = document.createElement('h4');
+  projectHeading.innerHTML = project.title;
+  section.appendChild(projectHeading);
+  // <p>
+  const projectDesciption = document.createElement('p');
+  projectDesciption.textContent = project.description;
+  section.appendChild(projectDesciption);
+  // skills
+  const projectSkills = document.createElement('ul');
+  const skills = project.technologies;
+  // looping the skills
+  for (let i = 0; i < skills.length; i += 1) {
+    const projectSkillsListItem = document.createElement('li');
+    projectSkillsListItem.textContent = skills[i];
+    projectSkills.appendChild(projectSkillsListItem);
+    section.appendChild(projectSkills);
+  }
+  // <button>
+  const projectButton = document.createElement('button');
+  projectButton.classList = 'see-button';
+  projectButton.innerText = 'See project';
+
+  section.appendChild(projectButton);
+  // when clicked shows Id
+  projectButton.addEventListener('click', () => {
+    console.log(project.projectId);
+  });
+  return section;
+}
+
+for (let i = 0; i < projects.length; i += 1) {
+  const project = createProjectCard(projects[i]);
+  sectionCards.appendChild(project);
+}
+// POPUP-WINDOW
+const popup = {
+  title: 'Multi Post Stories',
+  techs: ['html', 'Bootstrap', 'Ruby on rails'],
+  imgSrc: './images/popup.png',
+  description:
+    "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+  livelink: 'https://have-samuel.github.io/mobile-ver',
+  sourceLink: 'https://github.com/Have-Samuel/mobile-ver',
+};
+
+// const popupWindow = document.querySelector('.popup1');
```

Comment:
- [ ] Please, kindly remove this piece of old commented code, so the repo gets cleaner 💪